tendrl/commons/flows/create_cluster/__init__.py

```diff
@@ -9,13 +9,32 @@
 from tendrl.commons.flows import utils
 from tendrl.commons.flows.create_cluster import ceph_help
 from tendrl.commons.flows.create_cluster import gluster_help
+from tendrl.commons.flows.exceptions import FlowExecutionFailedError
 from tendrl.commons.flows.import_cluster.ceph_help import import_ceph
 from tendrl.commons.flows.import_cluster.gluster_help import import_gluster
 from tendrl.commons.objects.job import Job
 
 
 class CreateCluster(flows.BaseFlow):
     def run(self):
+        # Check if clusre name contains space char and fail if so
+        if ' ' in self.parameters['TendrlContext.cluster_name']:
+            Event(
+                Message(
+                    priority="info",
```

Comment:
This info goes as job update because `job_id` and `flow_id` both are present and so would be listed under job updates. Actual error would happen due to `FlowExecutionFailedError`